### ISSUE TITLE
DOI not recognized when uploading occurrence data

### ISSUE THREAD

**Issue by aneriswanepoel** [kartoza/django-bims#2781]
FBIS is not recognizing the DOI when uploading occurrence data. Error message:
![image](https://user-images.githubusercontent.com/55532903/134163294-50060fcb-68a4-4b2b-ae6f-e561c9569851.png)

@dimasciput

**Comment by dimasciput** [comment]
@Fanevanjanahary let me handle this one

**Comment by dimasciput** [comment]
fixed on testing @aneriswanepoel

**Comment by aneriswanepoel** [comment]
@dimasciput thanks, working now. Will you let me know when it is working on Live too?

**Comment by helendallas** [comment]
@aneriswanepoel Please can you check if this isi working on live

**Comment by aneriswanepoel** [comment]
> @aneriswanepoel Please can you check if this isi working on live

no not working yet

**Comment by aneriswanepoel** [comment]
> @aneriswanepoel So you tried to upload this data and you are still getting an error message?? On live??

yeah tried it on live but still got the same error message. It worked on testing though so maybe it hasn't been rolled onto live yet?
